Lodestar build broken due to `export = _exports`

Built `index.d.ts` file break a Typescript build with 

```
node_modules/libp2p-tcp/dist/src/index.d.ts:2:1 - error TS2309: An export assignment cannot be used in a module with other exported elements.
2 export = _exports;
  ~~~~~~~~~~~~~~~~~~
Found 1 error.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Link to a CI run with error above https://github.com/ChainSafe/lodestar/pull/3059/checks?check_run_id=3494564063#step:5:52

The file `index.d.ts` is:

```
lodestar$ cat node_modules/libp2p-tcp/dist/src/index.d.ts
declare const _exports: any;
export = _exports;
export type Multiaddr = import('multiaddr').Multiaddr;
export type Connection = import("libp2p-interfaces/src/connection/connection");
export type Upgrader = import('libp2p-interfaces/src/transport/types').Upgrader;
export type Listener = import('libp2p-interfaces/src/transport/types').Listener;
export type Socket = import('net').Socket;
//# sourceMappingURL=index.d.ts.map
```

Blocks https://github.com/ChainSafe/lodestar/pull/3059

--- a/package.json
+++ b/package.json
@@ -41,7 +41,7 @@
   "types": "dist/src/index.d.ts",
   "devDependencies": {
     "@types/debug": "^4.1.5",
-    "aegir": "^33.2.0",
+    "aegir": "^35.0.3",
     "it-pipe": "^1.1.0",
     "libp2p-interfaces": "^1.0.0",
     "libp2p-interfaces-compliance-tests": "^1.0.0",

--- a/src/index.js
+++ b/src/index.js
@@ -151,4 +151,6 @@ class TCP {
   }
 }
 
-module.exports = withIs(TCP, { className: 'TCP', symbolName: '@libp2p/js-libp2p-tcp/tcp' })
+const TCPWithIs = withIs(TCP, { className: 'TCP', symbolName: '@libp2p/js-libp2p-tcp/tcp' })
+
+exports = module.exports = TCPWithIs

